config/routes.rb

```diff
@@ -8,14 +8,14 @@
    match "/404", to: "errors#not_found", via: :all
    match "/500", to: "errors#internal_server_error", via: :all
 
-   get '/jobs/:job_id/applies/rejected', to: 'applies#rejected', as: 'rejected'
-   get '/jobs/:job_id/applies/accepted', to: 'applies#accepted', as: 'accepted'
+   get '/jobs/:job_id/job_applications/rejected', to: 'job_applications#rejected', as: 'rejected'
+   get '/jobs/:job_id/job_applications/accepted', to: 'job_applications#accepted', as: 'accepted'
```

Comment:
move all these routes into the resources

Reply:
the routes where added in to resources